frontend/src/views/cancellations.js

```diff
@@ -5,34 +5,59 @@ import airplane from "../images/airplane.png";
 import { Button } from "../components/ui";
 import ValueCancelation from "../components/value-cancelation";
 import { useRound } from "../selectors";
+import { navigate } from "@reach/router";
+import Header from "../components/header";
+import { useSetCurrentGame } from "../action-hooks";
 
 function FlightDetails() {
   // TODO: WIP animation of values
   const game = useRound();
-  const seatsStatus = game.cancellations - game.overbookingNumber;
-  const totalRevenue = game.totalRevenue / 1000;
-  const [cancellation, setCancellations] = React.useState(game.cancellations);
-  const [overbookingCost, setOverbookingCost] = React.useState(
-    seatsStatus < 0 ? (seatsStatus * game.overbookingCost) / -1000 : 0
+  const {
+    cancellations,
+    overbookingNumber,
+    totalRevenue,
+    overbookingCost,
+    underageCost
+  } = game.games[game.currentGame];
```

Comment:
Could you explain how this works?

Reply:
we are using destructuring to get elements of a specific game from the games object in the state.